db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/ExecutorUtils.java

```diff
@@ -48,23 +48,34 @@ private static boolean awaitTermination(ExecutorService executor, Duration timeo
 	}
 
 	public static ThreadFactory defaultThreadFactoryWithPrefix(String prefix) {
-		return new PrefixingDefaultThreadFactory(prefix);
+		return new PrefixingDefaultThreadFactory(prefix, false);
+	}
+
+	public static ThreadFactory defaultThreadFactoryWithPrefix(String prefix, boolean useDaemonThreads) {
+		return new PrefixingDefaultThreadFactory(prefix, useDaemonThreads);
 	}
```

Comment:
I can't seem to find any references to this method (where daemon=true)? Was there meant to be?